scripts/publish.py

```
@@ -0,0 +1,270 @@
+        description="Markdown ドラフトをはてなブログに下書き投稿する",
+    )
+    parser.add_argument(
+        "draft_file",
+        type=Path,
+        help="投稿する Markdown ドラフトファイルのパス",
+    )
+    parser.add_argument(
+        "--env-file",
+        type=Path,
+        default=Path(".env"),
+        help="環境変数ファイルのパス（既定: .env）",
```

Ruff (0.15.12)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)